rules/docker-sonic-mgmt.mk

```diff
@@ -3,4 +3,4 @@ DOCKER_SONIC_MGMT = docker-sonic-mgmt.gz
 $(DOCKER_SONIC_MGMT)_PATH = $(DOCKERS_PATH)/docker-sonic-mgmt
 $(DOCKER_SONIC_MGMT)_DEPENDS += $(SONIC_DEVICE_DATA) $(PTF)
 SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_MGMT)
-SONIC_STRETCH_DOCKERS += $(DOCKER_SONIC_MGMT)
+SONIC_BUSTER_DOCKERS += $(DOCKER_SONIC_MGMT)
```

Comment:
Can we remove it? using the latest docker distribution?

Reply:
test good.